### PR TITLE
fix(orb-ui): #162 matching agent displayed instead of group details

### PR DESCRIPTION
**Pull request by joaoguilherme2003**

### CODE REVIEW

File: ui/src/app/shared/components/orb/policy/policy-datasets/policy-datasets.component.html [hunk 1]
```diff
@@ -39,7 +39,7 @@
 </ng-template>
 
 <ng-template #groupTemplateCell let-i="index" let-row="row" let-value="value">
-    <button (click)="onOpenGroupDetails(row?.agent_group)"
```
Comment (etaques) [inline ui/src/app/shared/components/orb/policy/policy-datasets/policy-datasets.component.html:42]: I saw you created new function to perform the action.. Did you removed this **onOpenGroupDetails** function or still being used?

**Review by etaques — CHANGES_REQUESTED** [review]

File: ui/src/app/shared/components/orb/policy/policy-datasets/policy-datasets.component.html (hunk 1 above)
Comment (etaques) [reply ui/src/app/shared/components/orb/policy/policy-datasets/policy-datasets.component.html:42]: yes, I strongly believe we removing code that are not being used would be good to don't confuse other programmers that can be working on the code futurely.

**Review by gpazuch — APPROVED** [review]
LGTM :+1:

**Review by etaques — APPROVED** [review]
LGTM

**Pull request merged**